src/cms/cms.js

```diff
@@ -3,6 +3,9 @@ import uploadcare from "netlify-cms-media-library-uploadcare";
 import cloudinary from "netlify-cms-media-library-cloudinary";
 import { ColorControl, ColorPreview } from "netlify-cms-widget-colorpicker";
 
+// import relevant styles here for previews
+import "../components/all.scss";
```

Comment:
In your rework it's of value to _not_ do styles like this in a single file

Is convenient for dev's but comes with a perf cost for users